trymito.io/pages/infrastructure-integration-python-tool.tsx

```diff
@@ -0,0 +1,212 @@
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+            <section className={classNames(pageStyles.background_card, titleStyles.title_card)}>
+                <h1 className={titleStyles.title}>
+                    Integrate Mito into your Enterprise Python Infrastructure
+                </h1>
+
+                <p className={titleStyles.description}>
+                    Let your analysts use custom Python functions and code snippets directly through the Mito spreadsheet
+                </p>
+                    
+                <div className={homeStyles.cta_button_and_video_spacer}>
+                    <CTAButtons variant='contact' align='center'/>
+                </div>
+                    
+                <div id='video'>
+                    <video className={homeStyles.video} autoPlay loop disablePictureInPicture playsInline webkit-playsinline="true" muted>
+                        <source src="/custom_imports.mp4" />
+                    </video>
+                </div>
+            </section>
+
+            <section>
+                <div className={classNames(pageStyles.subsection, pageStyles.subsection_column, 'center', titleStyles.title)}>
+                    <h2>
+                        Custom Python functions shouldn&apos;t sit unused in a GitHub repo. Make them accessible in the Mito spreadsheet.
+                    </h2>
+                    <p>
+                        Analysts that don&apos;t know how to use Python functions, use spreadsheet functions instead.
+                    </p>
+                </div>
+                <div className={classNames(pageStyles.subsection, pageStyles.subsection_justify_baseline)}>
+                    <div className={textImageSplitStyles.functionality_text}>
+                        <h2>
+                            Configurable <span className='text-highlight'>Database Queries</span>
+                        </h2>
+                        <p> 
+                            Expose configurable database imports through the Mito UI. 
+                            Mito automatically detects the parameters and generates an intuitive UI for non-technical users. 
+                        </p>
+                        <p>
+                            Once the data is in Mito, analysts can use all of Mito&apos;s transformations to complete their analysis. 
+                        </p>
+                    </div>
+                    <div className={classNames(textImageSplitStyles.functionality_media, textImageSplitStyles.functionality_media_supress_bottom_margin)}>
+                        <Image src={'/configurable-database-queries.png'} alt='Configurable Database Queries' width={650} height={358} layout='responsive'/>
```

Comment:
For this image, can we:
1. Make the Python code screen shot above the UI
2. Have the Python code show the function definition and the UI

I think showing the generated code is less cool / unique than showing the generated UI - which is the point of this bullet anyway.